package.json

```diff
@@ -121,7 +130,111 @@
                 "command": "extension.randomRgbColor",
                 "title": "Random: Color (RGB)"
             }
-        ]
+        ],
+        "menus": {
+            "editor/context": [
+                {
+                    "command": "extension.resetSeed",
```

Comment:
Can we remove the `resetSeed` option from the context menu? I doubt that it will be that much useful 😅

Reply:
Fixed